deployment/terraform/cdn.tf

```diff
@@ -0,0 +1,80 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  origin {
+    domain_name = "origin.${var.r53_public_hosted_zone}"
+    origin_id   = "originAlb"
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
+    }
+  }
+
+  enabled         = true
+  is_ipv6_enabled = true
+  http_version    = "http2"
+  comment         = "${var.project} (${var.environment})"
+
+  price_class = "${var.cloudfront_price_class}"
+  aliases     = ["${var.r53_public_hosted_zone}"]
+
+  default_cache_behavior {
+    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originAlb"
+
+    forwarded_values {
+      query_string = true
+      headers      = ["*"]
+
+      cookies {
+        forward = "all"
+      }
+    }
+
+    compress               = false
+    viewer_protocol_policy = "redirect-to-https"
+    min_ttl                = 0
+    default_ttl            = 0
+    max_ttl                = 300
+  }
+
+  ordered_cache_behavior {
+    path_pattern     = "static/*"
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originAlb"
+
+    forwarded_values {
+      query_string = false
+
+      cookies {
+        forward = "none"
+      }
+    }
+
+    compress               = true
+    viewer_protocol_policy = "redirect-to-https"
+    min_ttl                = 0
+    default_ttl            = 300
+    max_ttl                = 300
+  }
+
+  logging_config {
+    include_cookies = false
+    bucket          = "${aws_s3_bucket.logs.id}.s3.amazonaws.com"
+  }
+
+  restrictions {
+    geo_restriction {
+      restriction_type = "none"
+    }
+  }
+
```

Comment:
It looks like the `aws_cloudfront_distribution` resource supports `tags` now. I think it would be valuable to add the standard ones here too.